set-up-point-cloud/syskit-point-subscriptions.md

```diff
@@ -46,39 +47,32 @@ The **Overview section (1)** provides insight into general information on the st
 * **Status (3)** of your Syskit Point license
 * **Next renewal date (4)** of your Syskit Point subscription or expiration date of your free trial
 * **Licenses (5)** the number of licenses included in your Syskit Point
-* **Purchased Plan (6)** for your subscription
+   * The number of licensed users affects the pricing of your Syskit Point subscription
+   * A minimum of 100 licenses need to be purchased for your Syskit Point subscription. For more details on this, take a look at the [**Licensed Users Count article**](../licensing-activation/licensed-users-count.md). 
+* **Purchased Plan (6)** for your subscription as well as the amount of Audit Logs Storage years you have available
   * This shows as Trial while your free trial period is still active
-* Next to **Purchased Plan** you can click on the **Buy Now button (7)** to purchase your Syskit Point subscription or manually click the **Subscriptions section (8)** on the left side of the screen under Overview
+
+* Next to Purchased Plan, you can click on the **Buy Now button (7)** to purchase your Syskit Point subscription or manually click the **Subscriptions section (8)** on the left side of the screen under Overview
+
 
 ![Syskit Point Subscription - Overview Screen](../.gitbook/assets/purchasing-syskit-point-subscriptions-overview.png)
 
-The **Subscription section (1)** is where you can purchase your subscription and select the specific details that suit your needs.
```

Comment:
ovaj tekst (cijela uputa o purchasingu) se nije trebala micati niti mijenjati screenshot. 
tu smo još u purchase sekciji i u stanju gdje customer nije prvi put kupio subscription - prijašnji screenshot je bio dobar jer se u tom slučaju prikazuje inicijalni buy screen kao što je ovdje i bilo opisano.